bin/main.rb

```diff
@@ -1,59 +1,59 @@
 #!/usr/bin/env ruby
+require_relative '../lib/game_logic.rb'
+require_relative '../lib/player.rb'
 
-# Welcome to Tic Tac Toe Game
-puts 'Welcome to Tic Tac Toe Game'
-
-# Display game board
+def show_board(board)
+  puts '-------------'
+  puts "| #{board[0]} | #{board[1]} | #{board[2]} |"
+  puts '-------------'
+  puts "| #{board[3]} | #{board[4]} | #{board[5]} |"
+  puts '-------------'
+  puts "| #{board[6]} | #{board[7]} | #{board[8]} |"
+  puts '-------------'
+end
```

Comment:
## Not in the main file 
-  This method should come from the `game_logic.rb` file, not in the main file
  - [ ] Thus, please move to `game_logic.rb` file,

Reply:
We had to implement this in the main file since the instructions are to have `puts` and `gets` only in this file and nowhere else